tutorials/Tutorial5_Evaluation.py

```diff
@@ -142,17 +136,16 @@ def tutorial5_evaluation():
         p.add_node(component=eval_reader, name="EvalAnswers", inputs=["QAReader"])
         results = []
 
-        for q, l in q_to_l_dict.items():
+        for l in labels:
             res = p.run(
-                query=q,
+                query=l.question,
                 top_k_retriever=10,
-                labels=l,
+                labels={"reader": l, "retriever": l},
```

Comment:
are the keys here still correct? I thought we wanted to change them to the node names, i.e. ESRetriever and QAReader in this pipeline